raiden/blockchain/state.py

```diff
@@ -77,7 +78,8 @@ def get_channel_state(
         open_transaction=open_transaction,
         close_transaction=close_transaction,
         settle_transaction=settle_transaction,
-        is_light_channel=is_light_channel
+        is_light_channel=is_light_channel,
+        both_participants_are_light_clients=both_participants_are_light_clients
```

Comment:
This must be changed in a future to an enum that represents: 

1: full nodes channel
2: full node with lc channel
3: lc - lc channel

now it will take a lot of time and refactor